pytorchfi/error_models.py

```diff
@@ -307,15 +307,15 @@ def random_neuron_single_bit_inj(pfi_model, layer_ranges):
     pfi_model.set_conv_max(layer_ranges)
 
     batch = random_batch_element(pfi_model)
-    (conv, C, H, W) = random_neuron_location(pfi_model)
+    (layer, C, H, W) = random_neuron_location(pfi_model)
 
     return pfi_model.declare_neuron_fi(
-        batch=batch,
-        conv_num=conv,
-        c=C,
-        h=H,
-        w=W,
-        function=pfi_model.single_bit_flip_signed_across_batch,
+        batch=[batch],
+        layer_num=[layer],
+        c=[C],
+        h=[H],
+        w=[W],
+        function=pfi_model.single_bit_flip_signed_across_batch, #TODO Support multiple error models via list
```

Comment:
after merge we can make this a task so then there are less `todo` in codebase :)

Reply:
Added #56 !